site/style.css

```
@@ -31,7 +48,9 @@ body {
   background: var(--bg);
   color: var(--fg);
   font:
-    16px/1.75 -apple-system,
+    16px/1.75 "Pretendard Variable",
+    "Pretendard",
```

Stylelint (17.14.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)